platform/broadcom/sonic-platform-modules-cel/silverstone/modules/switchboard.c

```diff
@@ -1431,7 +1431,7 @@ static int smbus_access(struct i2c_adapter *adapter, u16 addr,
         goto Done;
     }
 
-#if 0 /* 100 kHz */
+#if 1 /* 100 kHz */
```

Comment:
Revert the clock to high-speed, unless it's unlikely to be the root cause, and we actually have this enabled in ICOS, and I never got any issues for this